app/Services/Northstar/Northstar.php

```diff
@@ -71,7 +71,7 @@ public function verifyUser($credentials)
     public function getUserSignups($id, $campaigns)
     {
         $response = $this->get('signups', [
-            'user' => $id,
+            'users' => $id,
```

Comment:
well users can work for both 1 id, or both. remember you had me change this in drupals?